Sources/SWBUtil/Path.swift

```diff
@@ -69,18 +69,45 @@ public struct Path: Serializable, Sendable {
     /// The system path separator.
     #if os(Windows)
     public static let pathSeparator = Character("\\")
-    public static let pathSeparatorUTF8 = UInt8(ascii: "\\")
```

Comment:
Using an inline computed var instead of a stored let saves a little memory and a little indirection. Is it actually a relevant amount? Not really.